yarn-project/key-store/src/test_key_store.ts

```diff
@@ -292,4 +292,17 @@ export class TestKeyStore implements KeyStore {
     }
     return Promise.resolve(Fr.fromBuffer(publicKeysHashBuffer));
   }
+
+  public async addPublicKeysForAccount(
```

Comment:
Before this PR key store was used only for accounts which are registered as accounts in PXE and not only as recipient. See [this code](https://github.com/AztecProtocol/aztec-packages/blob/587973ff94368b5ae6247f04c26fd0ef6a933b51/yarn-project/pxe/src/pxe_service/pxe_service.ts#L228).

I think it's a good idea to keep this separation.

I am fine with just adding here a TODO(#5834) though.